src/services/userServices.js

```diff
@@ -23,7 +23,20 @@ export default class UserServices {
  * @returns {object} return the user's data
  */
   static async getUserByEmail(email) {
-      const data = await models.User.findOne({ where: { email, } });
-      return data;
+    const data = await models.User.findOne({ where: { email } });
+    return data;
+  }
+
+  /**
+   * @name LoginUser
+   * @description Interacts with model to sign in a user
+   * @param { string } email the user's email
+   * @returns { object } returns the user's data
+   */
+  static async loginUser(email) {
+    const data = await this.getUserByEmail(email)
```

Comment:
smart to use the already defined method in the class, but in this case don't you think you can just use that method directly in your controller since it essentially does the same thing, which is return a single user by email

Reply:
What is the need to recreate the `getUserByEmail` method

Reply:
I've refactored it